server-protocol.md

```diff
@@ -205,14 +215,12 @@ nameplates for that whole time.
 
 The `allocate` command allocates a nameplate (the server returns one that is
 as short as possible), and the `allocated` response provides the answer.
-Clients can also send a `list` command to get back a `nameplates` response
-with all allocated nameplates for the bound AppID: this helps the code-input
-tab-completion feature know which prefixes to offer. The `nameplates`
-response returns a list of dictionaries, one per claimed nameplate, with at
-least an `id` key in each one (with the nameplate string). Future versions
-may record additional attributes in the nameplate records, specifically a
-wordlist identifier and a code length (again to help with code-completion on
-the receiver).
+
+There is a `list` command (with the answer message being `nameplates`) intended
+for the use-case of listing currently in-use nameplates for user input
+auto-completion purposes. However, this feature could trivially be used to
+disrupt the service, therefore servers now **must not** disclose any information
+about in-use nameplates. Clients should expect always getting an empty response.
```

Comment:
I think it would be better (overall) to make clear what are unimplemented future features/directions versus current behavior. (That is, currently this is an operator decision, not a protocol decision .. although we may decide to deprecate this / add the wordlist thing -- so maybe link to a ticket here)

Reply:
The only "must" here (currently) is that clients cannot depend on getting any information: they may (always) get an empty response to this request.

Reply:
I see two options here: either we say servers **should not** disclose in-use nameplates, in which case the respective server configuration its default value flipped. Or we make it a **must not**, in which case it may be deprecated/removed. Either way, nothing really changes for clients unless they (wrongly) assume to always get a response with content.

One more question would be whether the server should give an error instead of an empty response, although in the end it does not matter that much.

Reply:
I'm not opposed to changing the behavior in the future, but _right now_ this is up to the operator. So these words should reflect that (possibly linking to a server ticket describing the changed / future behavior).

Both currently-deployed public servers allow `LIST`.